src/utils/mathUtils.ts

```diff
@@ -89,6 +93,13 @@ export module Utils {
       return range;
     }
 
+    /**
+     * Returns the square of the distance between two points
+     *
+     * @param p1
+     * @param p2
+     * @return dist(p1, p2)^2
```

Comment:
Incorrect docs here. It should be:
```
/**
 * @param {number} p1
 * @param {number} p2
 * @returns {number}
 */
```

However, I think this method's name is very self-explanatory, so it'd be better just not to have any docs on this method at all.